README.md

```diff
@@ -20,12 +23,13 @@ The sample code generates two reports:
    | `daysSinceLastPull` | The number of days since the repository was last pulled |
    | `lifecyclePolicyText` | The lifecycle policy text of the repository |
 
-   The contents of this report help to identify the repositories that are not being used and can be deleted to reduce costs. The contents also allows to identify which repositories have the most images, the most heaviest ones, and the ones without lifecycle policies; which impacts the cost of the repository. Last but not least, this report allows to see which repositories have security scan enabled in a consolidated way.
+The contents of this report help to identify the repositories that are not being used and can be deleted to reduce costs. The contents also allows to identify which repositories have the most images, the most heaviest ones, and the ones without lifecycle policies; which impacts the cost of the repository. Last but not least, this report helps identifying which repositories have security scan enabled in a consolidated way.
 
-   > [!CAUTION]
-   > The totalSize(MB) field sums the size of the images in the repository which may provide you with insights about repositories that are consuming more storage and therefore may be candidates for optimization. However, **this field cannot be used to determine the storage cost for each repository**. Amazon ECR does not apply duplicate charges for container layers, which means that if you have several images with the same layer, it will only be counted once for pricing purposes. To obtain the exact storage cost for each repository, you would need to identify the different layers within all images of the repository and calculate the cost based on the size of the unique layers combined.
+> **[!CAUTION]**
+> The totalSize(MB) field sums the size of the images in the repository which may provide you with insights about repositories that are consuming more storage and therefore may be candidates for optimization. However, **this field cannot be used to determine the storage cost for each repository**. Amazon ECR does not apply duplicate charges for container layers, which means that if you have several images with the same layer, it will only be counted once for pricing purposes. To obtain the exact storage cost for each repository, you would need to identify the different layers within all images of the repository and calculate the cost based on the size of the unique layers combined.
```

Comment:
Use the exact text from blogpost.

Reply:
Its the same one. Just the formatting was wrong.

Reply:
Oh ok I understand. I thought you meant compared with the previous version.